app/controllers/comments_controller.rb

```diff
@@ -52,7 +52,7 @@ def edit
   # POST /comments
   # POST /comments.json
   def create
-    if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
+    if rate_limiter.limit_by_action(:comment_creation)
```

Comment:
what if we use the old and tested technique of raising the error and rescueing from it in a central way? Same as we do in the API.

https://github.com/thepracticaldev/dev.to/blob/db7aa197fa75bbe15ce73df4fa6bfea5627d6a06/app/controllers/api/v0/api_controller.rb#L20-L22

Afer all here we just render and bail

Same goes for the other "if rate limit; then bail" examples :D